src/helm/proxy/clients/openai_client.py

```diff
@@ -187,8 +192,91 @@ def do_it():
             embedding=embedding,
         )
 
+    @staticmethod
+    def _get_tokenizer_name(tokenizer: str) -> str:
+        return "/".join(tokenizer.split("/")[1:])
```

Comment:
Let's just do `tokenizer.split("/")[1]`; I don't think we need the join.

Reply:
Done.